exercises/julia-1-a/.meta/config.json

```diff
@@ -0,0 +1,162 @@
+{
+    "solution_files": ["example.jl", "example2.jl"],
+    "test_file": "runtests.jl",
+    "tests": [
+        {
+            "test": "Finished game where X won via colum victory: \"XOO\nX  \nX  \"",
```

Comment:
`colum` should be `column` (there are a couple of other places with this minor typo).